docs/source/essentials/00-queries.mdx

```diff
@@ -1,28 +1,27 @@
+Fetching data in a predictable, type-safe way is one of the core features of Apollo Android. In this guide, you'll learn how to query a GraphQL endpoint and use the result in your application.
 
-This page assumes some familiarity with building GraphQL queries. If you'd like a refresher, we recommend [reading this guide](http://graphql.org/learn/queries/) and practicing [running queries in GraphiQL](https://graphql.github.io/swapi-graphql/).
-Since Apollo queries are just standard GraphQL, anything you can type into the GraphiQL query explorer can also be put into `.graphql` files in your project.
+## Prerequisites
 
-The following examples assume that you've already set up Apollo Android for your Android/Kotlin application. Read our [getting started](../#get-started) guide if you need help with either of those steps.
+This page assumes some familiarity with building GraphQL queries. For a refresher, we recommend [reading this guide](http://graphql.org/learn/queries/) and practicing [running queries in Apollo Sandbox](https://studio.apollographql.com/sandbox/explorer?endpoint=https%3A%2F%2Fswapi-graphql.netlify.app%2F.netlify%2Ffunctions%2Findex&explorerURLState=N4IgJg9gxgrgtgUwHYBcQC4QEcYIE4CeABAOIIoBiAlgDZwDORwAOkkUQIY03V2Mtt2RAGa0GTVkKEoqKGgklCAvopVIlIADQgAbhzxUOAI3n0MIEEqA).
 
-Apollo-android takes a schema and a set of `.graphql` files and uses these to generate code you can use to execute queries and access typed results.
+> Because Apollo Android uses standard GraphQL syntax, anything query you can run in Sandbox can also be put into `.graphql` files in your project.
 
-> All `.graphql` files in your project (or the subset you specify as input to `apollo-codegen` if you customize the script you define as the code generation build phase) will be combined and treated as one big GraphQL document. That means fragments defined in one `.graphql` file are available to all other `.graphql` files for example, but it also means operation names and fragment names have to be unique and you will receive validation errors if they are not.
+This page also assumes that you've already set up Apollo Android for your Android/Kotlin application. For help with setup, see the [getting started guide](../#get-started).
 
-## Creating queries 
+## Defining
 
-Queries are represented as instances of generated classes conforming to the `Query` protocol. Constructor arguments can be used to define query variables if needed.
-You pass a query object to `ApolloClient#query(query)` to send the query to the server, execute it, and receive results.
+In Apollo Android, each query you execute is represented as an instance of a generated class that conforms to the `Query` protocol. To generate these classes, we first need to define the GraphQL operations we want to execute.
 
-For example, if you define a query called `GetHero`:
+Let's say we define a GraphQL query named `HeroQuery`, like so:
 
-```graphql
+```graphql:title=hero-query.graphql
```

Comment:
We don't have really strong suggestions for naming (but I think we should, same for fragments). For the time being, I'd do:

```suggestion
```graphql:title=HeroQuery.graphql
```